ls1intum/Artemis
# `Programming exercises`: Fix test case issues not showing immediately

### Checklist
#### General
#### Client

### Motivation and Context
@MarkusPaulsen noticed that on the initial edit view (directly after pressing the edit button) the problem statement always shows "no test case issues". The first parsing is only done once the problem statement gets edited.

### Description
Additionally call analyzeTasks() in ngOnInit()

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Programming Exercise

1. Make sure that the problem statement has test case issues (e.g. wrong tests mentioned in tasks or tests not mentioned at all)
2. Reload the page
3. Check that the issue indicator directly shows the issues

### Review Progress

#### Code Review
#### Manual Tests

### Screenshots
![grafik](https://user-images.githubusercontent.com/26540346/197207485-fcbeb3d4-97d2-4f9c-97c0-bcc4780780a8.png)
